### ISSUE TITLE
Search Deflector Option Not Showing Up In Windows Options

### ISSUE THREAD

**Issue by GiulioMicheletti** [spikespaz/search-deflector#238]
## Bug Description

I've installed the program normally, I tried using it and it didn't work, I searched the wiki, tried to change the default app by protocol but it didn't show up.

## How to Reproduce

**Browser**: Opera Stable

**Search Engine:** DuckDuckGo

## Screenshots

![Capture](https://user-images.githubusercontent.com/95318975/144138093-4b034470-b722-4d05-a4fa-fd335e8c1125.PNG)

<kbd>Alt</kbd><kbd>PrntScr</kbd> will store a screenshot of the active window in your clipboard. Pressing <kbd>Ctrl</kbd><kbd>V</kbd> will paste the picture at the current cursor position when editing this issue.

## Environmental Details

| Windows Version | `Example: 1903` |
| --- | --- |
| Build Number | `Example: 18362.10015` |

**Comment by bsclifton** [comment]
Per the release notes here for Windows 11 build 22494:
https://blogs.windows.com/windows-insider/2021/11/12/releasing-windows-11-build-22000-346-to-beta-and-release-preview-channels/
![image](https://user-images.githubusercontent.com/4733304/141597957-45a595c4-e7a4-43c4-a706-65fd6081f5e7.png)

It seems devs at Microsoft considered overriding `microsoft-edge` a bug. If that's true, I'm not sure why they approved Search Deflector? 🤷 

Also see https://www.howtogeek.com/768727/microsoft-calls-firefoxs-browser-workaround-improper-will-block-it/

**Comment by GiulioMicheletti** [comment]
Curse you Microsoft!!!!!!😡

**Comment by zcorn2017** [comment]
Is there any workaround for it now?

**Comment by powercrusher** [comment]
came across this today, contacted support and they didnt really help solve this, im hoping a workaround does come around because i hate edge so much

**Comment by spikespaz** [comment]
@powercrusher If I missed your email, I apologize! I try to respond to them but I don't always faithfully check my email and sometimes they get missed.

I will put here what I have been telling other users:

Microsoft continues to break the `MICROSOFT-EDGE` protocol and a new version of Search Deflector will be released when a workaround is implemented. They have declined two of my updates on the Microsoft Store, claiming that the app has bugs and doesn't work on all platforms (which is true, because of them).

There are already several solutions that I have in mind, but because this requires a complete re-implementation, I have to find the time to do it, and I have other things to worry about. This project doesn't really make any funds for me, though I do appreciate the donations, they are not enough to buy my time to update this program. Therefore, I will get to it when I get to it.

Additionally, I would be developing this in a virtual machine as I no longer actually use Windows (which is pretty much directly because of Microsoft's behavior). Therefore I would be relying on users entirely to ensure that there are no bugs and that this works as expected (which there are enough of you to do so, but not without shipping broken code to find out).

**TL;DR:** Fixing these issues will require a lot of work. Version 2.0.0 is coming, but has no ETA, and I work on it when I have spare time in the [`rewrite-rust`](https://github.com/spikespaz/search-deflector/tree/rewrite-rust) branch.
